image/tools/lib/component/resources.sh

```diff
@@ -39,10 +39,11 @@ function backup_resource {
             echo '---' > /tmp/${type}.yaml
             for obj in $(oc get ${type} -n ${ns} | tr -s ' ' | cut -d ' ' -f 1 |  tail -n +2); do
                 echo '-' >> /tmp/${type}.yaml
```

Comment:
This line is no longer required.

Reply:
and `echo '---' > /tmp/${type}.yaml` also.
/tmp/${type}.yaml doesn't have any real content besides dashes.